script/Arcueid/Day 6/06_00_ARC06_2_2.txt

```diff
@@ -242,97 +287,114 @@ Let me get that covered at once."
 }
 [c78559d614aeeb0c6d41a49e903a5701fa870535]{
 -- Page 237, Offset 14485.
 -- その後、カカトをテーピングで固定し、包帯を巻いて治療は完了した。
-After that, she tapes my heel in place and wraps it in a bandage, thus completing the treatment.
+That accomplished, she tapes my heel in place and wraps it in a bandage, thus completing the treatment.
 }
 [606cdb3ec24faa2eb98a9049bf8ccc8ae68e5b8f]{
 -- Page 237, Offset 14486.
 -- ……本当に見事な手際だった。
-...It really was a brilliant job.
+...She really did a brilliant job.
 }
 [72845258f898c8809b61a756f1c6efb828176940]{
 -- Page 237, Offset 14487.
 -- 有間の家にいた頃お世話になっていた医院のじじいにも引けを取らない。この人、黙っていれば名医なんじゃないのか？
-It's even comparable to the old man from the clinic that I owe a great deal to back at the Arima house. This person might even be an excellent physician if only she could keep her mouth shut.
+It's on par with the care I received from the old man from the clinic, who took care of me back when I was living with the Arimas.
+ She'd probably be an excellent physician if she could only keep her mouth shut.
 }
 [6ac59120e4f3060d02042430c8f5ad15d81f07b5]{
 -- Page 238, Offset 14488.
 -- 「ありがとうございます。おかげで助かりました」
-"Thank you very much. I feel better now thanks to you."
+"Thank you very much. I feel a lot better now thanks to you."
 }
 [1d1d95ce0e45595f743d1dc8f38dc986803e85e7]{
 -- Page 238, Offset 14489. Mods: @k, @e.
 -- 「いいのいいの。アタシ、しょせん無免かつ専門外だし？　本職はカガク者だし？
-"Don't mention it. Besides, I'm not licensed and this is outside my expertise. I'm mainly a scientist after all.
+"Don't mention it. Besides, I'm not even licensed and this is outside my area of expertise. I'm really more of a scientist.
 }
 [4cc43f5eb89dc608704245bb8f7d1cf2dfce9903]{
 -- Page 238, Offset 14490. Glued. Mods: @x.
 -- 志貴チャン、博士って呼んでくれたし？」
-Not to mention that Shiki-chan called me a professor."
+// Leading space for glue
+ Shiki-chan called me a professor after all."
 }
 [acde0b55ba9a0e7317e101d55d915d134e432b99]{
 -- Page 239, Offset 14491.
 -- 博士と呼ばれた事がよほど嬉しいらしい。
-It appears that she is very happy to be called professor.
+She seems to be over the moon about the fact that I called her 'professor'.
 }
 [6e1dee1203851baf461755a9af989b1e61aea463]{
 -- Page 239, Offset 14492.
 -- ……この様子では、今まで秋葉たちに呼ばれた事はないのだろう。
-...Based on this situation, it seems that Akiha and the others have never called her that up until now.
+...Based on her reaction, I assume that Akiha and the others have never referred to her that way before.
 }
 [c353ee0b7ca1d515cf0b891950f0a26a2d8d2dfd]{
 -- Page 240, Offset 14493.
 -- 「けどぉ、志貴チャン、ずいぶんと体に無茶させたみたいねぇ。人体の運動性、その限界に挑んだ感じ？
-"But, Shiki-chan, it looks like you pushed your body through quite a bit. It's as if you were testing the limits of your human mobility.
+// Dictionary defines 運動性 as motility, which is a bit awkward as-is. Mildly sideways translated.
+"But you know, Shiki-chan,
+ it really looks like you've put your body through the wringer.
+ It's as though you were trying to surpass the limits of what's physically possible for a human.
```

Comment:
'maneuverability of the human body' for 人体の運動性

Reply:
Yeah, I originally tried to fit that in but just could not get it to read well. Since it's just 運動 + 性 I figured that the general "physical possibility" was _close enough_. If you can come up with a smooth sentence that uses 'maneuverability of the human body' happy to switch though